examples/gds-sessions.ipynb

```diff
@@ -0,0 +1,392 @@
+    "\n",
+    "Now that we have created a session, let's list all our sessions to see what that looks like"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sessions.list()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding a dataset\n",
+    "\n",
+    "We assume that the configured AuraDB instance is empty.\n",
+    "We will add our dataset using standard Cypher.\n",
+    "\n",
+    "In a more realistic scenario, this step is already done, and we would just connect to the existing database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_query = \"\"\"\n",
+    "  CREATE\n",
+    "    (dan:Person {name: 'Dan',     age: 18, experience: 63, hipster: 0}),\n",
+    "    (annie:Person {name: 'Annie', age: 12, experience: 5, hipster: 0}),\n",
+    "    (matt:Person {name: 'Matt',   age: 22, experience: 42, hipster: 0}),\n",
+    "    (jeff:Person {name: 'Jeff',   age: 51, experience: 12, hipster: 0}),\n",
+    "    (brie:Person {name: 'Brie',   age: 31, experience: 6, hipster: 0}),\n",
+    "    (elsa:Person {name: 'Elsa',   age: 65, experience: 23, hipster: 1}),\n",
+    "    (john:Person {name: 'John',   age: 4, experience: 100, hipster: 0}),\n",
+    "    \n",
+    "    (apple:Fruit {name: 'Apple',   tropical: 0, sourness: 0.3, sweetness: 0.6}),\n",
+    "    (banana:Fruit {name: 'Banana', tropical: 1, sourness: 0.1, sweetness: 0.9}),\n",
+    "    (mango:Fruit {name: 'Mango',   tropical: 1, sourness: 0.3, sweetness: 1.0}),\n",
+    "    (plum:Fruit {name: 'Plum',     tropical: 0, sourness: 0.5, sweetness: 0.8})\n",
+    "  \n",
+    "  CREATE\n",
+    "    (dan)-[:LIKES]->(apple),\n",
+    "    (annie)-[:LIKES]->(banana),\n",
+    "    (matt)-[:LIKES]->(mango),\n",
+    "    (jeff)-[:LIKES]->(mango),\n",
+    "    (brie)-[:LIKES]->(banana),\n",
+    "    (elsa)-[:LIKES]->(plum),\n",
+    "    (john)-[:LIKES]->(plum),\n",
+    "  \n",
+    "    (dan)-[:KNOWS]->(annie),\n",
+    "    (dan)-[:KNOWS]->(matt),\n",
+    "    (annie)-[:KNOWS]->(matt),\n",
+    "    (annie)-[:KNOWS]->(jeff),\n",
+    "    (annie)-[:KNOWS]->(brie),\n",
+    "    (matt)-[:KNOWS]->(brie),\n",
+    "    (brie)-[:KNOWS]->(elsa),\n",
+    "    (brie)-[:KNOWS]->(jeff),\n",
+    "    (john)-[:KNOWS]->(jeff);\n",
+    "\"\"\"\n",
+    "\n",
+    "# making sure the database is actually empty\n",
+    "assert gds.run_cypher(\"MATCH (n) RETURN count(n)\").squeeze() == 0\n",
+    "\n",
+    "# let's now write our graph!\n",
+    "gds.run_cypher(data_query)\n",
+    "\n",
+    "gds.run_cypher(\"MATCH (n) RETURN count(n) AS nodeCount\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Projecting Graphs\n",
+    "\n",
+    "Now that we have imported a graph to our database, we can project it into our GDS Session.\n",
```

Comment:
should we explain the remote projection query a bit?

Reply:
Done.